index.html

```diff
@@ -736,6 +745,10 @@ <h2 class="display-4 text-center" id="data_header">Loading data...</h2>
           </ul>
           <!-- Hunter Survival -->
           <ul class="tc-box" id="tc_hunter_survival" hidden>
+            <li><span class="translate_survival">Survival</span> Guides
```

Comment:
Please update all three hunter spec guide links to the following:
- MM links to icy-veins and wowhead
- BM links to icy-veins only
- SV links to icy-veins only